app/src/main/java/com/kamron/pogoiv/PokeInfoCalculator.java

```diff
@@ -29,7 +29,10 @@ public PokeInfoCalculator(String[] namesArray, int[] attackArray, int[] defenceA
     }
 
     public Pokemon get(int number) {
-        return pokedex.get(number);
+        if (number >= 0){
+            return pokedex.get(number);
+        }
+        return null;
```

Comment:
or if you want to keep this it'd be nice to annotate the method as `@Nullable` so callers can see it can return `null` in their IDE